src/main/java/qna/domain/Answer.java

```diff
@@ -109,10 +96,11 @@ public void setDeleted(boolean deleted) {
     public String toString() {
         return "Answer{" +
                 "id=" + id +
-                ", writerId=" + writerId +
-                ", questionId=" + questionId +
+                ", writerId=" + writer.toString() +
+                ", questionId=" + question.toString() +
```

Comment:
```suggestion
                ", question=" + question +
```

이런식으로 사용하셔도 됩니당~

(양방향 순환 참조만 주의)